ROMFS/px4fmu_common/init.d-posix/airframes/1022_uuv_bluerov2_heavy

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+#
+# @name BlueROV2 Heavy Configuration
+#
+
+. ${R}etc/init.d/rc.uuv_defaults
+
+if [ $AUTOCNF = yes ]
+then
+	#Set data link loss failsafe mode (0: disabled)
+	param set NAV_DLL_ACT 0
+
+	# disable circuit breaker for airspeed sensor
+	param set CBRK_AIRSPD_CHK 162128
+
+fi
+
+set MAV_TYPE 12
+param set MAV_TYPE ${MAV_TYPE}
```

Comment:
Can you move this to https://github.com/PX4/Firmware/blob/master/ROMFS/px4fmu_common/init.d/rc.vehicle_setup#L191, the same as for the other vehicle types?

Reply:
Does the same also apply for `60002_uuv_bluerov2_heavy`?